src/main/controllers/service/ServiceSearchResultsController.ts

```diff
@@ -0,0 +1,32 @@
+import { Response } from 'express';
+import { FactRequest } from '../../interfaces/FactRequest';
+import autobind from 'autobind-decorator';
+import { FactApi } from '../../utils/FactApi';
+
+@autobind
+export class ServiceSearchResultsController {
+
+  constructor(
+    private readonly api: FactApi
+  ) { }
+
+  public async get(req: FactRequest, res: Response) {
+    const courtsData = await this.api.courtsInServiceAreas(req.params.service, req.params.serviceArea);
+    const serviceAreaData = await this.api.getServiceArea(req.params.service, req.params.serviceArea);
+    for( const court of courtsData ){
+      if( court.catchment === 'national' ){
+        req.i18n.getDataByLanguage(req.lng)['service-results'].nameOfCourt = req.i18n.getDataByLanguage(req.lng)['service-results'].nameOfCourt
```

Comment:
i think it might be best to store this in a object and do stuff like data.nameOfCourt like in searchResults controller for name known.

Reply:
Have changed this now